combinations/hash.tsv

```diff
@@ -226,3 +226,4 @@ python=2.7,pysam=0.17.0,flask=1.1.2,cython=0.29.14,numpy=1.16.6,scipy=1.2.1,matp
 samclip=0.4.0,samtools=1.15
 bowtie2=2.4.5,samtools=1.15,pigz=2.6
 longshot=0.4.1,samtools=1.15
+python=3.9,matplotlib=3.5.1,pandas=1.3.5,r-sys=3.4,regex=2021.11.10,scipy=1.7.3,click=8.0.4
```

Comment:
if you do not need paython 3.9 you do not need to specify it

Reply:
you need R in this containers as well? Can this step not be done in a separate container?

Reply:
@bgruening I updated as you comments

`biopython=1.78,pandas=1.3.5,matplotlib=3.5.1,typer=0.4.1,xlsxwriter=1.4.3`